### DISCUSSION TITLE
I have a forum!

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#390] [Announcements]
https://jdev.eu.org/forum
discuss catalyst pls

**Comment by bredcat999** [comment]
why does it say there have been 6 registrations from my IP address in the last 24 hours

**Reply by jdev082** [reply]
@bredcat999 Just some configuration issues. Try again, it should work.

**Reply by bredcat999** [reply]
same message unfortunately

**Reply by jdev082** [reply]
@bredcat999 fixed 100%
